route/build/src/network_topo.f90

```diff
@@ -655,6 +660,14 @@ SUBROUTINE REACH_LIST(&
   structSEG(iRch)%var(ixSEG%totalArea)%dat(1) = structSEG(iRch)%var(ixSEG%basArea)%dat(1) + &
                                                 structSEG(iRch)%var(ixSEG%upsArea)%dat(1)
 
+  ! ---------- define good basins ---------------------------------------------------------
+
```

Comment:
i think need to define what the good basin is here.   
"goodBasin"  == Drainage area at the bottom of the reach is greater than threshold, "verySmall" value.

Reply:
Fixed